catalyst/utils/checkpoint.py

```diff
@@ -84,11 +84,27 @@ def save_checkpoint(
     is_best: bool = False,
     is_last: bool = False,
     special_suffix: str = "",
+    saver_fn: Callable = torch.save,
```

Comment:
just `save_fn`? btw, what do you think about simple `is_xla_model` flag? so
```
if is_xla_model:
   xla_model_save(checkpoint, filename)
else:
   torch.save(checkpoint, filename)
```
?